src/applications/personalization/dashboard/utils/date-formatting/timezone.js

```diff
@@ -84,13 +84,13 @@ export function getTimezoneNameFromAbbr(abbreviation) {
 }
 
```

Comment:
Worth mentioning: for this set of utils the function `getTimezoneByFacilityId` uses `id.substr(0, 3)` in the final return.

That [string method is deprecated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr), so we should move to substring()

Also when I ran coverage, I saw that final return path was not covered.

Adding a test like this should be able to cover it:
```
  it('should search for a facility ID with extra characters at end, trim them, and return matching timezone', () => {
    expect(getTimezoneByFacilityId('436ZZZZZZ')).to.equal('America/Denver');
  });
  ```